src/script.js

```diff
@@ -176,7 +176,7 @@
 			e.preventDefault();
 		});
 
-		saveHtmlBtn.addEventListener('click', function (e) {
+		function save_file() {
```

Comment:
This project is written with camelCase convention. So `saveFile` instead. Also now that this is a named function, would be better off outside the `init` function. Put it alongside `init` function.